packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -34,6 +35,7 @@ export interface ApplyFieldOverrideOptions {
   fieldOptions: FieldConfigSource;
   replaceVariables: InterpolateFunction;
   theme: GrafanaTheme;
+  timeZone?: TimeZone;
```

Comment:
This change is kind of unrelated... but I discovered it while trying to test the time formatting.